src/main/java/com/github/igorperikov/hollow/HollowAPIGeneratorUtility.java

```diff
@@ -42,8 +42,8 @@ public static HollowAPIGenerator createHollowAPIGenerator(
                 .withBooleanFieldErgonomics(properties.useBooleanFieldErgonomics)
                 .reservePrimaryKeyIndexForTypeWithPrimaryKey(properties.reservePrimaryKeyIndexForTypeWithPrimaryKey)
                 .withHollowPrimitiveTypes(properties.useHollowPrimitiveTypes)
-                .withVerboseToString(properties.useVerboseToString)
-                .withClassPostfix(properties.classPostfix);
```

Comment:
isn't it removes functionality from the plugin?

Reply:
Gah! Didn’t manage to switch forks correctly. Will try it again, it’s supposed to be added back after an if not blank test.

Reply:
Done: [Lines 58-60](https://github.com/IgorPerikov/hollow-maven-plugin/pull/16/files#diff-5be73f72a08e5671a738ad28ffadcc8dR58-R60)

Reply:
this one looks fixed, thanks

I'll merge and release it a bit later